featuretools/primitives/base/primitive_base.py

```diff
@@ -51,3 +57,23 @@ def get_function(self):
 
     def get_filepath(self, filename):
         return os.path.join(config.get("primitive_data_folder"), filename)
+
+    def get_args_string(self):
+        def valid(arg):
+            error = '"{}" must be attribute of {}'
+            assert hasattr(self, arg.name), error.format(arg.name, self.__class__.__name__)
+            is_positional_or_keyword = arg.kind == arg.POSITIONAL_OR_KEYWORD
+            is_same_type = isinstance(getattr(self, arg.name), type(arg.default))
+            default = is_same_type and arg.default == getattr(self, arg.name)
+            return is_positional_or_keyword and not default
+
+        string = OrderedDict()
```

Comment:
does this need to be an `OrderedDict`? why not just make it a list

Reply:
No, doesn't need to be. I will make into list.